css/main.css

```diff
@@ -1,3 +1,201 @@
 /* Add your custom CSS here */
+@import url('https://fonts.googleapis.com/css?family=PT+Sans&display=swap');
+body {
+    font-family: 'PT Sans', sans-serif;
+    background: rgb(219, 218, 218);
+    color: sienna;
+}
+/* custom gridfor about-us */
+.row-2 {
```

Comment:
the bootstrap grid classes should already provide this as "row" - see the comment made on your index.html